### PR TITLE
MOMNG-141 Create State Error types

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
The following state error types should be included across the USSD and IVR applications
Invalid Date
Invalid Selection
Invalid Number
The copy for each of these is included in the copy spreadsheet: https://docs.google.com/spreadsheets/d/1ZeWPISUYcDqOOei5wraQGRI_9xcvvM6w7jwnjQqWx6o/edit#gid=995314643
On USSD, when a state error occurs, the error state copy will display + the question will be repeated. e.g.
Invalid selection:
Sorry, invalid selection. Please choose your language...
On IVR, when a state error occurs, the error state recording will play directly followed by the question . e.g.
Invalid selection:
The option you selected is not valid. Please choose your language...

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review my progress.  I made changes to all non-voice apps first, but will proceed with the voice apps after you've :1'ed what's done so far.

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review.  changes made to all non-voice apps

File: src/train_ussd_public.js [hunk 1]
```diff
@@ -17,101 +17,71 @@ go.app = function() {
...
-            "state_msg_language":
-                $("Sorry not a valid input. What language would this person like to receive these messages in?"),
-            "state_optout_reason":
-                $("Sorry not a valid input. Please tell us why you no longer want to receive messages so we can help you further."),
-            "state_loss_subscription":
-                $("Sorry not a valid input. We are sorry for your loss. Would you like to receive a small set of free messages from Hello Mama that could help you in this difficult time?"),
-            "state_optout_receiver":
-                $("Sorry not a valid input. Who would you like to stop receiving messages?"),
+        var state_error_types = {
+            "invalid_selection": "Sorry, invalid option. ",
```
Comment (imsickofmaps) [inline src/train_ussd_public.js:83]: Missing `$` round the error text

Comment (imsickofmaps) [inline src/train_ussd_public.js:83]: Making changes to this returns the good old `object Object` issue 👎  - leave it with me

**Comment by NicolaasDieBaas** [comment]
opened a separate issue for creating state error types for IVR apps (https://github.com/praekelt/mama-ng-jsbox/issues/143)

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**